Sources/StreamChat/Database/DTOs/CurrentUserDTO_Tests.swift

```diff
@@ -62,7 +62,8 @@ class CurrentUserModelDTO_Tests: XCTestCase {
             Assert.willBeEqual(mutedUserIDs, Set(loadedCurrentUser?.mutedUsers.map(\.id) ?? []))
             Assert.willBeEqual(payload.devices.count, loadedCurrentUser?.devices.count)
             Assert.willBeEqual(payload.devices.first?.id, loadedCurrentUser?.devices.first?.id)
-            // TODO: Teams
+            Assert.willBeEqual(payload.teams, loadedCurrentUser?.user.teams?.map(\.id))
+            Assert.willBeFalse(payload.teams.isEmpty)
```

Comment:
Do we need the insurance the test data (payload) is valid?

AFAIU it will fail if someone breaks the` CurrentUserPayload.dummy` by removing the `teams: ["GREEN", "RED"]` inside. 

Wouldn't it be nicer to update the `dummy` to get `teams: [String]` as input param and explicitly provide ["GREEN", "RED"] when we create a payload in **test_currentUserPayload_isStoredAndLoadedFromDB**?

Reply:
✅

Reply:
I'd say we also don't need `Assert.willBeFalse(payload.teams.isEmpty)` since we are testing the input data here. If it fails it's a developer mistake, not related to `CurrentUserDTO`